react-app/src/components/property/PropertyDialog.tsx

```diff
@@ -324,18 +430,75 @@ export const PropertyNetBookValueEditDialog = (props: IPropertyNetBookValueEditD
   );
   const netBookFormMethods = useForm({
     defaultValues: { Fiscals: [] },
+    mode: 'onBlur',
   });
+  const currentYear = new Date().getFullYear();
+  const years = [currentYear];
+  const defaultValues = years.map((year) => ({
+    FiscalYear: year,
+    Value: 0,
+    EffectiveDate: undefined,
+    FiscalKeyId: 0,
+    PropertyType: initialValues?.PropertyTypeId || null,
+    Id: initialValues?.Id || null,
+  }));
```

Comment:
Same sort of issue as above. `years` will only ever be an array of 1, so why not:
```suggestion
  const defaultValues = [{
    FiscalYear: currentYear,
    Value: 0,
    EffectiveDate: undefined,
    FiscalKeyId: 0,
    PropertyType: initialValues?.PropertyTypeId || null,
    Id: initialValues?.Id || null,
  }];
```